pkg/util/procfs/procfs_test.go

```diff
@@ -56,3 +59,17 @@ func TestContainerNameFromProcCgroup(t *testing.T) {
 	procCgroupInvalid := "devices:docker/kubelet\ncpuacct:pkg/kubectl"
 	verifyContainerName(procCgroupInvalid, "", true, t)
 }
+
+func TestPidOf(t *testing.T) {
+	if runtime.GOOS == "darwin" || runtime.GOOS == "windows" {
+		t.Skipf("not supported on GOOS=%s", runtime.GOOS)
+	}
+	// Let's look for the init process
+	pids, _ := PidOf("init")
+	// test-unit-integration environment is slightly different
+	if len(pids) == 0 {
```

Comment:
is there a better way to detect this environment? I don't like this heuristic based guess.